features/admin.push-providers.v1/components/push-provider-card.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Card from "@oxygen-ui/react/Card";
+import CardContent from "@oxygen-ui/react/CardContent";
+import Typography from "@oxygen-ui/react/Typography";
+import {
+    CustomAttributeInterface,
+    ExtensionTemplateListInterface,
+    ResourceTypes
+} from "@wso2is/admin.template-core.v1/models/templates";
+import { ExtensionTemplateManagementUtils } from "@wso2is/admin.template-core.v1/utils/templates";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import classnames from "classnames";
+import React, { FunctionComponent, MouseEvent, ReactElement, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import "./push-provider-card.scss";
+import { PushProviderConstants } from "../constants/push-provider-constants";
+import { PushProviderTemplateFeatureStatus } from "../models/templates";
+
+/**
+ * Props for the push provider card component.
+ */
+export interface PushProviderCardPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Callback function triggered upon clicking on the push provider card.
+     *
+     * @param e - Click event.
+     */
+    onClick?: (e: MouseEvent<HTMLDivElement>) => void;
+    /**
+     * Details needed to render the push provider card.
+     */
+    template: ExtensionTemplateListInterface;
+
+    selected: boolean
+}
+
+const PushProviderCard: FunctionComponent<PushProviderCardPropsInterface> = ({
+    template,
+    onClick,
+    selected,
+    ["data-componentid"]: componentId = "push-provider-card"
+}: PushProviderCardPropsInterface): ReactElement => {
+
+    const { t } = useTranslation();
+
+    const featureStatus: PushProviderTemplateFeatureStatus = useMemo(() => {
+        if (!template?.customAttributes
+            || !Array.isArray(template?.customAttributes)
+            || template?.customAttributes.length === 0) {
+            return false;
+        }
+
+        const property: CustomAttributeInterface = template?.customAttributes?.find(
+            (property: CustomAttributeInterface) =>
+                property?.key === PushProviderConstants.FEATURE_STATUS_ATTRIBUTE_KEY
+        );
+
+        return property?.value;
+    }, [ template ]);
+
+    const resolveRibbonLabel = (featureStatus: PushProviderTemplateFeatureStatus): string => {
```

Comment:
Let's use the new config model here.

Reply:
Please refer to https://github.com/wso2/identity-apps/blob/master/features/admin.server-configurations.v1/components/governance-connector-grid.tsx#L259

Reply:
Here we are using the feature status from the template.

https://github.com/wso2-extensions/identity-integration-ui-templates/blob/main/integrations/notification-providers/firebase/resources/info.json#L16

Reply:
Fixed with [b14a9b7](https://github.com/wso2/identity-apps/pull/7370/commits/b14a9b7d600ed5582f6b7db7d77a1c431ace7db8)